Simplenote/Classes/InterlinkViewController.swift

```diff
@@ -0,0 +1,184 @@
+import Foundation
+import UIKit
+
+
+// MARK: - InterlinkViewController
+//
+class InterlinkViewController: UIViewController {
+
+    /// Autocomplete TableView
+    ///
+    @IBOutlet private var tableView: UITableView!
+    @IBOutlet private var backgroundView: UIVisualEffectView!
+
+    /// Layout Constraints: Inner TableView
+    ///
+    @IBOutlet private var tableLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet private var tableTrailingConstraint: NSLayoutConstraint!
+    @IBOutlet private var tableTopConstraint: NSLayoutConstraint!
+    @IBOutlet private var tableHeightConstraint: NSLayoutConstraint!
+
+    /// KVO
+    ///
+    private var kvoOffsetToken: NSKeyValueObservation?
+
+    /// Interlink Notes to be presented onScreen
+    ///
+    var notes = [Note]() {
+        didSet {
+            tableView?.reloadData()
+        }
+    }
+
+
+    // MARK: - Overridden API(s)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupRootView()
+        setupBackgroundView()
+        setupTableView()
+        setupShadowLayer()
+    }
+}
+
+
+// MARK: - Public API(s)
+//
+extension InterlinkViewController {
+
+    /// Relocates the receiver so that it shows up around a given Keyword in a **Sibling TextView**
+    /// - Important: We'll start listening for Content Offset changes, and the UI will be automatically repositioned
+    ///
+    func anchorView(around keywordRange: Range<String.Index>, in siblingTextView: UITextView) {
+        refreshConstraints(keywordRange: keywordRange, in: siblingTextView)
+        startObservingContentOffset(in: siblingTextView)
+    }
+}
+
+
+// MARK: - Initialization
+//
+private extension InterlinkViewController {
+
+    func setupRootView() {
+        view.frame = UIScreen.main.bounds
```

Comment:
Do we need to set a frame here?

Reply:
We do need to do this manually, **but perhaps** it's best for the snippet that instantiates this VC to initialize it's view size. I'll relocate this over to the InterlinkProcessor, thank you!